frontend/README.md

```diff
@@ -39,3 +39,19 @@ If the standalone TypeScript plugin doesn't feel fast enough to you, Volar has a
    1. Run `Extensions: Show Built-in Extensions` from VSCode's command palette
    2. Find `TypeScript and JavaScript Language Features`, right click and select `Disable (Workspace)`
 2. Reload the VSCode window by running `Developer: Reload Window` from the command palette.
+
+## Setup Playwright for E2E Tests
+
+Playbook:
+
+- Make sure the backend runs
+- Make sure the frontend runs
+- Install Playwright
+- Let Playwright now where to connect to
+- Run the E2E Test
+
+```bash
+npx --yes playwright install --with-deps chrome firefox webkit
+
+E2E_BASE_URL=http://localhost:5173 npm run test:e2e
```

Comment:
You wouldn't need to pass that variable if you have the .env file. There is an example in the frontend folder